src/core/public/public.api.md

```diff
@@ -583,6 +583,7 @@ export interface DocLinksStart {
         readonly ml: Record<string, string>;
         readonly transforms: Record<string, string>;
         readonly visualize: Record<string, string>;
+        readonly apis: Record<string, string>;
```

Comment:
Is the intent for `apis` to be a general grouping of docs for all ES REST APIs? Asking because I want to make sure we know what belongs here moving forward.

Transforms have their own section, which feels a bit inconsistent, so I guess I'm wondering if we should consider something like this:
```
docLinks.links.transforms.putTransform
docLinks.links.indices.createIndex
```

Or perhaps group all ES docs generally like this:
```
docLinks.links.es.guides.transforms
docLinks.links.es.apis.transforms.putTransform
```

Either way I'm not too concerned about this ATM -- we really need to rethink how this service works anyway and it isn't difficult to change -- just thought I'd ask in case anyone has thoughts on it.

Reply:
At this point, my idea was that the apis grouping would contain reference material for APIs from any product or solution. In the near future, we're moving away from product-focused documentation, but we'll still have (hopefully auto-generated) reference pages for Kibana, Elasticsearch, and Cloud APIs, for example. I've added more examples in https://github.com/elastic/kibana/pull/86972.  If it's preferable to include the API links under feature groupings, I'm okay with that, though I'm not sure what feature we'd put thinks like the "create pipeline API" under.  An "ingest" grouping? Happy to consider whatever additional reorganization suggestions you have.

Reply:
> my idea was that the apis grouping would contain reference material for APIs from any product or solution. In the near future, we're moving away from product-focused documentation, but we'll still have (hopefully auto-generated) reference pages for Kibana, Elasticsearch, and Cloud APIs, for example.

Thanks, this is helpful context -- in that case the grouping makes a bit more sense to me.

> I'm not sure what feature we'd put things like the "create pipeline API" under. An "ingest" grouping? Happy to consider whatever additional reorganization suggestions you have.

Yeah I don't necessarily have an answer to this ATM either, I suppose I was mostly trying to get a picture of what you had in mind. 

It seems like it would be worth revisiting this discussion and re-evaluating how docLinks is structured further down the road, once the new docs are in place.